sdk/evaluation/azure-ai-evaluation/tests/unittests/test_evaluate_telemetry.py

```diff
@@ -141,11 +141,10 @@ def test_evaluator_start_telemetry(
         mock_trace_destination_to_cloud,
         mock_validate_trace_destination,
     ):
-        hate_unfairness = HateUnfairnessEvaluator(azure_ai_project=None)
 
         data = _get_file("evaluate_test_data.jsonl")
         evaluators = {
-            "hate_unfairness": hate_unfairness,
+            "hate_unfairness": HateUnfairnessEvaluator,
```

Comment:
Is this correct?

Reply:
this test does not use the evaluator since it gets passed to a dummy evaluate function that doesn't do anything with it. I can change it back, but the test still works regardless of passing the instance or the class

Reply:
Gotcha. Let's change it back otherwise it would cause confusion.